azurelinuxagent/ga/update.py

```diff
@@ -980,6 +985,34 @@ def _ensure_firewall_rules_persisted(dst_ip):
             message=msg,
             log_event=False)
 
+    @staticmethod
+    def _add_nonroot_tcp_firewallrule_if_not_enabled(dst_ip):
+
+        update_handler = get_update_handler()
+
+        # "-C" checks if the iptable rule is available in the chain
+        drop_rule = _get_firewall_drop_command("-w", "-C", dst_ip)
+        if not update_handler._check_if_iptable_rule_is_available(drop_rule):
+            return
+        else:
+            accept_non_root = _get_firewall_accept_command_nonroot_tcp("-w", "-C", dst_ip)
+            if not update_handler._check_if_iptable_rule_is_available(accept_non_root):
+                try:
+                    logger.info("Firewall rule to allow non root users to do tcp request to wireserver unavailable . Setting it now.")
```

Comment:
"to do tcp request to wireserver unavailable" -> "to do DNS TCP requests to wireserver does not exist"

Reply:
Addressed